plugins/processors/awsapplicationsignals/internal/normalizer/attributesnormalizer.go

```diff
@@ -8,7 +8,8 @@ import (
 	"strings"
 
 	"go.opentelemetry.io/collector/pdata/pcommon"
-	semconv "go.opentelemetry.io/collector/semconv/v1.18.0"
+	deprecatedsemconv "go.opentelemetry.io/collector/semconv/v1.18.0"
```

Comment:
do we still need the deprecated semconv?

Reply:
Yes, in fact, our ADOT SDK still emitting attributes using deprecated semconv.